Automattic/sensei
# Tweak course theme step text and images

Fixes #6226

### Changes proposed in this Pull Request

* Tweak the text at the top of the step.
* Update the screenshots and HTML with the fixed text for "Course".

### Testing instructions

Load the Course Theme step in the Setup Wizard, and ensure the text and screenshots look correct.

### Note

Adding these screenshots is a tradeoff between size and sharpness. We converted them to the `webp` format, which helps, and downsized them a bit so that they aren't too huge. I allowed a bit higher quality for the main screenshot so everything above the fold looks pretty sharp (but it's also the biggest image). The lower screenshots are a bit lower quality.

The `sensei-lms.zip` package for version 4.9.0 is 10.6 MB. The package sizes for this PR are shown below.

### Packages

From commit 5545b128b (old, with low-res images): [sensei-lms.zip](https://github.com/Automattic/sensei/files/10188592/sensei-lms.zip) (12.1 MB)

From commit 27da18ab6 (high-res images): [sensei-lms.zip](https://github.com/Automattic/sensei/files/10197384/sensei-lms.zip) (13.7 MB)
